.github/workflows/CI.yaml

```
@@ -70,10 +70,35 @@ jobs:
         pytest -v --cov=membrane_curvature --cov-report=xml --color=yes membrane_curvature/tests/
 
     - name: CodeCov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v2
       with:
         file: ./coverage.xml
         flags: unittests
         name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}
         fail_ci_if_error: True
 
+  latest_python:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        latest-python : ["3.11.0-rc.2"]
```

If you're only intending to use one version using a matrix might not be worth it here.
In any case py3.11 is now officially released.
```suggestion
        latest-python : ["3.11"]
```